src/lib/radio/radio.ts

```diff
@@ -149,6 +152,18 @@ export class MdRadioGroup implements AfterContentInit, ControlValueAccessor {
     }
   }
 
+  @Input()
+  get required(){
+    return this._required;
+  }
+
+  set required(required: boolean) {
+    this._required = required;
+
+    if (this._radios) {
+      this._radios.forEach(radio => radio.required = this.required);
+    }
```

Comment:
Not sure what you mean but I guess instead of this : `(value != null && value !== false) ? true : null;` simply use `coerceBooleanProperty`. I changed on the disabled as well. Pushed also